hack/ci/run-ipam-e2e-tests.sh

```diff
@@ -39,7 +39,37 @@ protokol --kubeconfig "$KUBECONFIG" --flat --output "$ARTIFACTS/logs/kubermatic"
 
 source hack/ci/setup-kubermatic-in-kind.sh
 
+echodate "Creating Hetzner preset..."
+cat << EOF > preset-hetzner.yaml
+apiVersion: kubermatic.k8c.io/v1
+kind: Preset
+metadata:
+  name: e2e-hetzner
+  namespace: kubermatic
+spec:
+  hetzner:
+    token: ${HZ_E2E_TOKEN}
+EOF
+retry 2 kubectl apply -f preset-hetzner.yaml
+
+echodate "Creating roxy-admin user..."
+cat << EOF > user.yaml
+apiVersion: kubermatic.k8c.io/v1
+kind: User
+metadata:
+  name: roxy-admin
+spec:
+  admin: true
+  email: roxy-admin@kubermatic.com
+  name: roxy-admin
+EOF
+retry 2 kubectl apply -f user.yaml
+
```

Comment:
Why do you create a roxy-admin user? The IPAM tests are headless, so there is no API/Dex.

Reply:
No need for it, indeed. I will remove it